scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -188,11 +201,12 @@ Changelog
 +-----------+------------------------+
 | Revision  | Changes                |
 +===========+========================+
-| 1.0       | Initial Draft           |
+| 1.1       | Initial Draft          |
```

Comment:
We should have this as a new line in the table, keeping the `1.0` row, and list what was changed in this version.